README.md

```diff
@@ -24,29 +52,65 @@ instructions, using the following parameters:
   * Set "Parameter key" as `worker_ip_addresses`, "Source type" as "Resource".
 
-### Create the Catalog Item
+### 2. Create the catalog item
 
-Follow the ["Create a catalog item"](https://servicedesk.surf.nl/wiki/display/WIKI/Create+a+catalog+item) instructions, using the following parameters:
+Follow the ["Create a catalog
+item"](https://servicedesk.surf.nl/wiki/display/WIKI/Create+a+catalog+item)
+instructions, using the following parameters:
 
 * In the "Available components" tab, choose the following components (in this order):
   * SRC-OS
   * SRC-CO
   * SRC-Ngnix
   * SRC-External plugin
   * the component created in the previous step
-* In the "Cloud providers" tab, add "SURF HPC Cloud cluster" with "Ubuntu 20.04" as OS and tick all available sizes.
+* In the "Cloud providers" tab, add "SURF HPC Cloud cluster" with "Ubuntu 20.04"
+  as OS and tick all available sizes.
 * In the "Ovewritable parameters" tab:
-  * For `co_passwordless_sudo`, set "Actions" to "Overwrite" and set "New parameter value" to `true`.
-  * For `num_nodes`, set "Actions" to "Make interactive", set "Label or question for workspace creator" to "Number of worker nodes", and set "Default parameter value" to `0`.
-  * For `dcache_token`, set "Actions" to "Make interactive", set "Label or question for workspace creator" to "dCache token: provide the macaroon to authenticate on the dCache storage", and set "Default parameter value" to `null`.
-  * For `environment_url`, set "Actions" to "Make interactive", set "Label or question for workspace creator" to "Conda environment file: provide the URL to a custom environment.yml (blank for default environment)", and set "Default parameter value" to `default`.
+  * For `co_passwordless_sudo`, set "Actions" to "Overwrite" and set "New
+    parameter value" to `true`.
+  * For `num_nodes`, set "Actions" to "Make interactive", set "Label or question
+    for workspace creator" to "Number of worker nodes", and set "Default
+    parameter value" to `0`.
+  * For `dcache_token`, set "Actions" to "Make interactive", set "Label or
+    question for workspace creator" to "dCache token: provide the macaroon to
+    authenticate on the dCache storage", and set "Default parameter value" to
+    `null`.
+  * For `environment_url`, set "Actions" to "Make interactive", set "Label or
+    question for workspace creator" to "Conda environment file: provide the URL
+    to a custom environment.yml (blank for default environment)", and set
+    "Default parameter value" to `default`.
+
+### 3. Create the workspace
+
+To create a workspace, you need to first create a storage, see [External storage
+volumes](https://servicedesk.surf.nl/wiki/display/WIKI/External+storage+volumes).
+Then, you can create a new workspace by following [Start a simple
+workspace](https://servicedesk.surf.nl/wiki/display/WIKI/Start+a+simple+workspace)
+and attach your storage to it. For the application, select the catalog item
+created in the previous steps.
```

Comment:
Actually, for I am not sure whether the storage will work with this component (the workspace includes multiple VMs, and I think a storage can be bound to a single VM only. To be tested, will raise an issue about this! 

```suggestion
You can create a new workspace by following [Start a simple
workspace](https://servicedesk.surf.nl/wiki/display/WIKI/Start+a+simple+workspace). 
For the application, select the catalog item
created in the previous steps. Follow the instructions and select parameters/provide input 
when requested.
```